lib/elixir/lib/task/supervisor.ex

```diff
@@ -77,6 +77,10 @@ defmodule Task.Supervisor do
           DynamicSupervisor.option()
           | DynamicSupervisor.init_option()
 
+  @typedoc "Options given to `async_stream` and `async_stream_nolink` functions"
+  @typedoc since: "1.17.0"
+  @type async_stream_option :: Task.async_stream_option() | {:shutdown, Supervisor.shutdown()}
```

Comment:
I'm not sure if this is confusing?
Might be simpler to duplicate the definition to avoid too much redirection, thoughts?

Reply:
I’m ok either way but duplicating also allows for diverging options in the future 👍

Reply:
Will follow this [precedent](https://github.com/elixir-lang/elixir/blob/v1.16.0/lib/elixir/lib/task/supervisor.ex#L76) and keep it as is for now.